### ISSUE TITLE
Adapt to data remodel

### ISSUE THREAD

**Issue by zbraniecki** [zbraniecki/covid19-timeseries#12]
The source that we're using here is remodeling their data: https://github.com/CSSEGISandData/COVID-19/issues/1250

```
* Three new time series tables will be added for the US. The first two will be the confirmed cases and deaths, reported at the county level. The third, number of tests conducted, will be reported at the state level. These new tables will be named time_series_covid19_confirmed_US.csv, time_series_covid19_deaths_US.csv, time_series_covid19_testing_US.csv, respectively.

* Changes to the current time series include the removal of the US state and county-level entries, which will be replaced with a new single country level entry for the US. The tables will be renamed time_series_covid19_confirmed_global.csv and time_series_covid19_deaths_global.csv, and time_series_covid19_testing_global.csv, respectively.

* The ISO code will be added in the global time series tables.

* The FIPS code will be added in the new US time series tables.

* We will no longer provide recovered cases.

* The current set of time series files will be moved to our archive folder, and the new files will be added to the current folder.

```

There is a number of nice things about it. Mainly per-state data (#5) and testing (#3) will be useful and the ISO codes will make #2 easier.

Unfortunately, due to inconsistency in data, we are going to lose "active" and/or "recovered" cases.

I'm not sure what to do about it, I think active timeseries for China is informative and shows the trajectory beyond escalation. I may look for an alternative source to continue keeping active/recovered cases even in an imperfect form.

**Comment by toqitahamid** [comment]
Do you have any plan to provide an API for "active" and "recovered" cases including the timeline?

**Comment by zbraniecki** [comment]
I don't plan to provide an API. This is an endpoint view of the data.

**Comment by zbraniecki** [comment]
I am most likely going to switch to #14 and use a new source with much more data and a very strong community behind it.

**Comment by zbraniecki** [comment]
Fixed in #14
